### DISCUSSION TITLE
The EA bot only opened once

### DISCUSSION THREAD

**Discussion by nahitua** [EA31337/EA31337#396] [General]
for EU pair ,first it starts and then it stops working I was waiting it to start but doesn't work till now what is the issue?

**Comment by kenorb** [comment]
- Can you check for any errors on chart or Journal log?
- Do you've Algo Trade enabled (on the top bar)?

**Reply by nahitua** [reply]
I can't see them on the chart it only says their is a wave or something unclear at some point and don't know what to do because it first started good with 0.01 lot size on my account after that i stopped it because my balance was 5k at that time and i was not satisfied with the lot size but after that i won't open any trade. 
And what is algo trade? How do i check if its enabled or not?

**Reply by kenorb** [reply]
Once you drag&drop into the chart, you should see smiley face in MT4 or hat icon in MT5 and bunch of text. If you see Elliott Waves popups, that means also it's working. If you're not seeing any errors, I think you're good. If it's still not trading, can you post the screenshot? Which platform and EA version you're using?

**Reply by nahitua** [reply]
okay here is the screenshot everything is fine but it won't open trades. 
![Screenshot (384)](https://github.com/EA31337/EA31337/assets/69083024/7ddcdae8-b543-43d2-b2ae-ac48fce7be65)
and am using the version 2.013

**Reply by nahitua** [reply]
also i keep getting thsese 
![Screenshot (386)](https://github.com/EA31337/EA31337/assets/69083024/fa1060d1-b9c3-4cc5-9b82-72efe8bf4349)

**Reply by kenorb** [reply]
Looks good.
Should trade if you're using the defaults settings, if you've changed your default params, the trading behaviour can be different.
Any warning/error messages in Experts/Journal log?

**Reply by nahitua** [reply]
No i haven't changed anything , and no error there , But it was once started when i firstly opened on the demo account then i was not satisfied with the lot size and while it was making profits I stopped it before hitting the TP , Does it affect? what shall i do? 
![Screenshot (388)](https://github.com/EA31337/EA31337/assets/69083024/ad89bb64-81df-4efd-9a52-85dc67180d79)
it says initializing only and no trade restrictions

**Reply by kenorb** [reply]
Difficult for me to tell. These messages are fine. So no errors at the end of Experts or Journal log? Everything looks fine, maybe the spread is too high for your broker? I don't have any extra indicator why EA would not trade (it should), and it looks fine for me (based on your screens). It's something to add.

**Reply by nahitua** [reply]
so what will you advice me to do, Should i delete and install again?

**Reply by kenorb** [reply]
Can you do backtesting for the last week to see if it trades as expected? Also, can you check whether EA Lite or Rider versions are trading fine?

**Reply by nahitua** [reply]
Okay let me do that and get back to you
